src/actions/guides/database/querying-deleting.cr

```diff
@@ -171,6 +171,15 @@ class Guides::Database::QueryingDeleting < GuideAction
     UserQuery.new.age(54)
     ```
 
+    In most cases, your value will come from a dynamic source like `params`.
```

Comment:
I’d maybe say in “some” cases since it really depends on the app. And maybe switch the sentences so it says “if the value...” since it may happen with non params and the most important part is the nil

Reply:
I originally wrote "some", and then changed it because I figured maybe it was edge case 😅  I've re-worded it now